yarl/_url.py

```diff
@@ -435,6 +450,21 @@ def raw_authority(self):
         """
         return self._val.netloc
 
+    def _get_default_port(self):
```

Comment:
And perhaps add typing:
```suggestion
    def _get_default_port(self) -> int | None:
```
(this probably needs a `from __future__ import annotations` at the top)

Reply:
Can we stick to:
```suggestion
    def _get_default_port(self) -> Optional[int]:
```
So, it still works when evaluating annotations at runtime, and just let pyupgrade change it when the time comes.